LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/UdpServer.cs

```diff
@@ -1,413 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace LoRaWan.NetworkServer
-{
-    using System;
-    using System.Globalization;
-    using System.Net;
-    using System.Net.Sockets;
-    using System.Security.Cryptography;
-    using System.Text;
-    using System.Threading;
-    using System.Threading.Tasks;
-    using LoRaTools;
-    using LoRaTools.ADR;
-    using LoRaTools.CommonAPI;
-    using LoRaTools.LoRaPhysical;
-    using LoRaTools.Utils;
-    using LoRaWan.NetworkServer.ADR;
-    using Microsoft.Azure.Devices.Client;
-    using Microsoft.Azure.Devices.Shared;
-    using Microsoft.Extensions.Caching.Memory;
-    using Microsoft.Extensions.Logging;
-    using Newtonsoft.Json;
-
-    /// <summary>
-    /// Defines udp Server communicating with packet forwarder.
-    /// </summary>
-    public sealed class UdpServer : IPacketForwarder, IDisposable
-    {
-        private const int PORT = 1680;
-        private readonly NetworkServerConfiguration configuration;
-        private readonly MessageDispatcher messageDispatcher;
-        private readonly LoRaDeviceAPIServiceBase loRaDeviceAPIService;
-        private readonly ILoRaDeviceRegistry loRaDeviceRegistry;
-        private readonly RandomNumberGenerator RndKeysGenerator = new RNGCryptoServiceProvider();
-
-        private IClassCDeviceMessageSender classCMessageSender;
-        private ModuleClient ioTHubModuleClient;
-        private volatile int pullAckRemoteLoRaAggregatorPort;
-        private volatile string pullAckRemoteLoRaAddress;
-        private UdpClient udpClient;
-
-        private Task<byte[]> GetTokenAsync()
-        {
-            var token = new byte[2];
-            this.RndKeysGenerator.GetBytes(token);
-            return Task.FromResult(token);
-        }
-
-        // Creates a new instance of UdpServer
-        public static UdpServer Create()
-        {
-            var configuration = NetworkServerConfiguration.CreateFromEnvironmentVariables();
-
-            var loRaDeviceAPIService = new LoRaDeviceAPIService(configuration, new ServiceFacadeHttpClientProvider(configuration, ApiVersion.LatestVersion));
```

Comment:
The `LoRaDeviceAPIService` seems to be unused now, can we remove it as well (same for `ServiceFacadeHttpClientProvider`)?

Reply:
LoRaDeviceAPIService is used in https://github.com/Azure/iotedge-lorawan-starterkit/blob/b152c4b6703c41fefbe4c6820ae5b181b5d9bd96/LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs#L36

Reply:
Yup, stupid mistake of mine, I was searching for constructor occurrences, not for occurrences of the type

Reply:
ServiceFacadeHttpClientProvider is used in https://github.com/Azure/iotedge-lorawan-starterkit/blob/b152c4b6703c41fefbe4c6820ae5b181b5d9bd96/LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationNetworkServerStartup.cs#L58